app/assets/javascripts/experiments/table.js

```diff
@@ -285,15 +306,51 @@ var ExperimnetTable = {
           return { ...params, ...{ filters: this.activeFilters } };
         }
       });
+
+      this.initProvisioningStatusPolling();
     });
   },
+  initProvisioningStatusPolling: function() {
+    let provisioningStatusUrls = $('[data-urls]').toArray()
```

Comment:
Maybe get provisioning rows and only URLs for this modules - `this.getUrls().provisioning_status